.github/workflows/build.yaml

```diff
@@ -37,7 +37,19 @@ jobs:
           version: edge
           static: ${{ matrix.os.static }}
       - run: npm install -g markdownlint-cli
+      - name: Restore rq cache
+        id: cache-rq
+        uses: actions/cache@v3
+        with:
+          path: ~/go/bin/rq
+          key: ${{ runner.os }}-${{ runner.arch }}-go-rq
```

Comment:
Worth adding a version? I guess we can worry about that when there's an rq version we want and it's not getting here because of the cache 😅

Reply:
Yeah... I did wonder about that. Added now.